fun main()
{
    try
    {
        val a = arrayOf(0, 2, 3)
        print(a[3])
    }
    catch (e:Exception)
    {
        print("You can not change the value of contant variable..")
    }

}
